src/components/CalltoActionButton.js

```diff
@@ -1 +1,43 @@
-//Gets passed a prop LinkTo that leads to the desired section, exists in two different styles based on its props (https://emotion.sh/docs/styled)
\ No newline at end of file
+import styled from "@emotion/styled"
+import React from "react"
+
+const ButtonLarge = styled.button`
+  border: solid 3px transparent;
+  border-radius: 100rem;
+  background-image: linear-gradient(
+      rgba(255, 255, 255, 0),
+      rgba(255, 255, 255, 0)
+    ),
+    linear-gradient(101deg, #78e4ff, #ff48fa);
+  background-origin: border-box;
+  background-clip: content-box, border-box;
+  padding: 0.5rem 3rem;
+  box-shadow: 2px 1000px 1px #111 inset;
+  color: white;
+  font-family: inherit;
+
+  & :hover {
+    animation: pulse 3s;
+    box-shadow: 0 0 0 2em rgba(#fff, 0);
+  }
+
+  @keyframes pulse {
+    0% {
+      box-shadow: 0 0 0 0 var(--gradient);
+    }
+  }
+`
+
+const ButtonSmall = styled(ButtonLarge)`
+  padding: 0.3rem 1rem;
+`
+
+function CalltoActionButton(props) {
+  if (props.size === "small") {
+    return <ButtonSmall>{props.children}</ButtonSmall>
+  } else {
+    return <ButtonLarge>{props.children}</ButtonLarge>
+  }
+}
+
+export default CalltoActionButton
```

Comment:
You have three components for your buttons. You can reduce it to a single button, if you use dynamic css:
https://emotion.sh/docs/styled#changing-based-on-props

```
 padding: ${(props) => props.size === 'small' ? '0.3rem 1rem' : '0.5rem 3rem'};
```

Reply:
For me, the name `CalltoActionButton` doesn't make sense (or I just don't understand 😄).
Isn't it a normal button with some special styling?

Reply:
Yes, it's a normal button which will scroll to a specific part of a page (functionality not added yet) where the user can take action.

Reply:
I implemented dynamic css and somehow it's rendering the same button twice, now...I'm probably missing something really stupid :D